omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/dex/DexSpec.groovy

```diff
@@ -60,18 +62,18 @@ class DexSpec extends BaseRegTestSpec {
         offerTx.propertyid == currencyOffered.getValue()
 
         where: "the currency identifier is either MSC or TMSC"
-        currencyOffered << [CurrencyID.MSC, CurrencyID.TMSC]
+        currencyOffered << [MSC, TMSC]
```

Comment:
The Omni Core API currently uses `"main"` and `"test"` as description for the ecosystem.

Maybe the extra `Ecosystem.` and `CurrencyID.` isn't a bad thing. It's very expressive, and if we adopt `MAIN` and `TEST`, then `MAIN` on it's own is probably a bit lonely.